src/vulnpy/trigger/nosqli.py

```diff
@@ -0,0 +1,48 @@
+from pymongo import MongoClient
+import os
+
+
+def create_mongo_client(**kwargs):
+    mongo_host = [os.environ.get("MONGODB_HOST", "localhost:27017")]
```

Comment:
do the `vulnpy` apps start up properly if the server doesn't have mongo running anywhere? I think it would be nice to have some graceful error handling in place for that case if pymongo doesn't already provide it